### ISSUE TITLE
Replace Issue

### ISSUE THREAD

**Issue by JosefJezek** [VFK/gulp-html-replace#23]
I have this code...
```js
return gulp.src('app/*.html')
    .pipe($.htmlReplace({
      vulcanized: {
        src: 'elements/elements.vulcanized.html',
        tpl: '<link rel="import" href="%s">'
      }
    }))
    .pipe(gulp.dest('dist'));
```
```html
<!-- build:css styles/vendor.css -->
  <!-- bower:css -->
  <!-- endbower -->
  <!-- endbuild -->

  <!-- build:css styles/main.css -->
  <link rel="stylesheet" href="styles/main.css">
  <!-- endbuild-->

  <!-- build:js bower_components/webcomponentsjs/webcomponents.min.js -->
  <script src="bower_components/webcomponentsjs/webcomponents.js"></script>
  <!-- endbuild -->

  <!-- build:vulcanized elements/elements.vulcanized.html -->
  <link rel="import" href="elements/elements.html">
  <!-- endbuild-->
</head>
<body class="polymer-theme" unresolved>
  <!-- build:js scripts/vendor.js -->
  <!-- bower:js -->
  <!-- endbower -->
  <!-- endbuild -->

  <!-- build:js scripts/app.js -->
  <script src="scripts/app.js"></script>
  <!-- endbuild-->
```
After
```html
<!-- build:css styles/vendor.css -->
  <!-- bower:css -->
  <!-- endbower -->

  <!-- build:css styles/main.css -->
  <link rel="stylesheet" href="styles/main.css">

  <!-- build:js bower_components/webcomponentsjs/webcomponents.min.js -->
  <script src="bower_components/webcomponentsjs/webcomponents.js"></script>

  <!-- build:vulcanized elements/elements.vulcanized.html -->
  <link rel="import" href="elements/elements.html">
</head>
<body class="polymer-theme" unresolved>
  <!-- build:js scripts/vendor.js -->
  <!-- bower:js -->
  <!-- endbower -->

  <!-- build:js scripts/app.js -->
  <script src="scripts/app.js"></script>
```

**Comment by VFK** [comment]
Hi @JosefJezek, sorry for the delay. First of all your HTML tag looks wrong. Look at [usage](https://github.com/VFK/gulp-html-replace#usage) section, you don't need anything other than task name. In fact this plugin will ignore such syntax because it thinks this is something for some other plugin :) So your HTML tag should look like this:
```html
<!-- build:vulcanized -->
```
I made some changes and published v1.4.4, it shouldn't wrongly remove end tags, thank you for the report.

And also, just to make sure, look at [options](https://github.com/VFK/gulp-html-replace#options) provided by this plugin if you didn't already. Since you're using it with some other replacement tools you may want to keep unused build tags or maybe don't want to remove those tags even if they're in your gulpfile. There are options for that.

**Comment by JosefJezek** [comment]
Ok, thank you.
